cfn-resources/maintenance-window/Makefile

```diff
@@ -1,11 +1,11 @@
-.PHONY: build test clean
+.PHONY: build debug clean
 tags=logging callback metrics scheduler
 cgo=0
 goos=linux
 goarch=amd64
 CFNREP_GIT_SHA?=$(shell git rev-parse HEAD)
 ldXflags=-s -w -X github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info -X github.com/mongodb/mongodbatlas-cloudformation-resources/version.Version=${CFNREP_GIT_SHA}
-ldXflagsD=-s -w -X github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug -X github.com/mongodb/mongodbatlas-cloudformation-resources/version.Version=${CFNREP_GIT_SHA}
+ldXflagsD=-X github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug -X github.com/mongodb/mongodbatlas-cloudformation-resources/version.Version=${CFNREP_GIT_SHA}
```

Comment:
any reason to remove -s and -w? is this is also used n the released bin it will increase its size

Reply:
This is for "debug" action in make, not used to release the resource. It's so we can debug and set breakpoints locally